ROADMAP.md

```diff
@@ -1,8 +1,19 @@
 # Roadmap
 
+## An even better tool for development environments
+
+Compose is a great tool for development environments, but it could be even better. For example:
+
+- [Compose could watch your code and automatically kick off builds when something changes.](https://github.com/docker/fig/issues/184)
```

Comment:
I _really_ like the idea of this feature, but happy to remove it if it's contentious.

@aanand WDYT?